hack/update-deps.sh

```diff
@@ -21,7 +21,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-go_update_deps "$@"
+go_update_deps "$@" --release v0.19
```

Comment:
do you need to run the script? I don't see any files changed besides this?
Are we using pkg, serving, and eventing 0.19?

Reply:
ohh I see eventing hasn't been cut yet..